libraries/botbuilder-ai-orchestrator/package.json

```diff
@@ -44,8 +44,7 @@
     "clean": "rimraf _ts3.4 lib tsconfig.tsbuildinfo",
     "lint": "eslint . --ext .js,.ts",
     "postbuild": "downlevel-dts lib _ts3.4/lib --checksum",
-    "test": "yarn build && nyc mocha tests/",
-    "test:compat": "api-extractor run --verbose"
+    "test": "yarn build && nyc mocha tests/"
```

Comment:
Note: I removed this in a few libraries that didn't have any `api.md` files. Should I add that back in so that they all have it?

Reply:
@stevengum what do you think?

Reply:
For context: the following libraries don't have the `api.md` files and so in this PR, I removed the `test:compat` script (which would fail if you tried to run them):

* botbuilder-ai-orchestrator
* botbuilder-azure-blobs
* botbuilder-azure-queues
* botbuilder-adaptive-testing

Reply:
I would imagine we want to run this command for all our public packages. If you add it to these packages (along with any other non-`private` packages) do things break?

Reply:
Not many things broke. I reverted this change and added `api-extractor` to all except the following packages:

**Covered by other libaries:**

* adaptive-expressions-ie11
* botbuilder-ai-luis
* botbuilder-ai-qna

**Internal/private:**

* bot-integration-tests
* botbuilder-repo-utils
* botbuilder-stdlib
* browser-functional-tests
* streaming-e2e-tests
* swagger
* teams-scenarios
* testbot
* testskills

**Issues with import statements ([will be fixed in the future](https://github.com/microsoft/rushstack/pull/1796))**

* botbuilder-lg
* botframework-connector